rules/windows/registry_event/sysmon_bypass_via_wsreset.yml

```diff
@@ -0,0 +1,30 @@
+title: UAC Bypass Via Wsreset
+id: 6ea3bf32-9680-422d-9f50-e90716b12a66
+status: experimental
+description: Unfixed method for UAC bypass from windows 10. WSReset.exe file associated with the Windows Store. It will run a binary file contained in a low-privilege registry.
+references:
+    - https://www.bleepingcomputer.com/news/security/trickbot-uses-a-new-windows-10-uac-bypass-to-launch-quietly
+    - https://lolbas-project.github.io/lolbas/Binaries/Wsreset
+    
```

Comment:
Hello @uchakin ! thank you for your contribution!

Could you please remove the empty line here?

Reply:
Hello! Thank you for the comment.
done

Reply:
thank you!